manager/integration/tests/test_upgrade.py

```diff
@@ -329,6 +329,19 @@ def test_upgrade(longhorn_install_method,
                               sspod_info['pod_name'],
                               sspod_info['data'])
 
+    # Create strict-local volume
```

Comment:
Can we update the test skeleton with strict-local volume

Reply:
Added. Thank you!